### PR TITLE
Mantis blade (new traitor item) (NOW with fixed conflicts)

### PR DESCRIPTION
**Pull request by enkidienne**
### *TRYING TO FIX CONFLICTS FROM #11348 
_if this works ill close the old one_

### General Documentation
Adds a new traitor item acquirable through any uplink for an expensive 16 TC. Your 16 TC gets you a fancy briefcase. Inside of it is 2 autodoc, 1 for each arm. Once used they will implant in your body a set of 2 retractable blades. These implants are stealthy and won't show on scans. The blades do 20 damage. Each attack, both blades will attack at once, dealing a total of 40 damage. The mantis blades are very noticeable once out so be careful. I have included a Nanostrasen version, but it is currently unobtainable.

**The** intent of your Pull Request
moar fun

Why is this change good for the game?
moar fun

### Wiki Documentation
The Mantis blades do 20 damage each and both attack at once.
The Nanostrasen version does 18 damage and both attack at once.

**Briefly describe your PR and the impacts of it, in layman's terms.**
New blades, traitor item. Blades do big damage when paired.

**What should players be aware of when it comes to the changes your PR is implementing?**
Beware of the people whose arms open to reveal dark and deadly blades.

**What general grouping does this PR fall under?**
Syndicate

**Are there any aspects of the PR that you would like us not to mention on the Wiki?**
no

**If there are any numerical values involved in your PR that will be relevant to a player, please note them here.**
Syndi blade: 20 damages
Paired syndi blades: 40 damages
NT blade: 18 damages
Paired NT blades: 36 damages

### Changelog
add: mantis blades

🆑

rscadd: mantis blade

/🆑
114496058-2e396580-9bed-11eb-8c6a-d1dddd51b9fe

114496095-38f3fa80-9bed-11eb-9630-f88e28f37139

on the right is the NT version and the left is syndie

ALL CREDIT GOES TO TG, I MERELY PORTED IT AND MAKE IT VIABLE.
original author: EdgeLordExe
tgstation/tgstation#55936

### CODE REVIEW

File: code/game/objects/items/mantis.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/obj/item/mantis/blade
+	name = "mantis blade"
+	desc = "Powerful inbuilt blade, hidden just beneath the skin. Singular brain signals directly link to this bad boy, allowing it to spring into action in just seconds."
```
Comment (Moggram) [inline code/game/objects/items/mantis.dm]: Grammar fix.

"A blade designed to be hidden just beneath the skin. The brain is directly linked to this bad boy, allowing it to spring into action."

File: code/modules/surgery/organs/augments_arms.dm [hunk 2]
```diff
@@ -233,3 +233,13 @@
 	name = "surgical toolset implant"
 	desc = "A set of surgical tools hidden behind a concealed panel on the user's arm."
 	contents = newlist(/obj/item/retractor/augment, /obj/item/hemostat/augment, /obj/item/cautery/augment, /obj/item/surgicaldrill/augment, /obj/item/scalpel/augment, /obj/item/circular_saw/augment, /obj/item/surgical_drapes)
+
+/obj/item/organ/cyberimp/arm/syndie_mantis
+	name = "G.O.R.L.E.X. mantis blade implants"
+	desc = "Modernized mantis blade designed coined by Tiger operatives, much sharper blade with energy actuators makes it a much deadlier weapon."
```
Comment (Moggram) [inline code/modules/surgery/organs/augments_arms.dm]: More grammar fix.

"Modernized mantis blade designed and coined by Tiger operatives. Energy actuators makes the blade a much deadlier weapon."

**Review by nesisknesis — APPROVED** [review]

**Comment by JamieD1** [comment]
@SomeguyManperson Balance review required

**Comment by JamieD1** [comment]
Please fix the Grammar and then this can be merged.

**Comment by enkidienne** [comment]
@JamieD1 done

**Comment by Hopekz** [comment]
@Marmio64 Can you provide more context on why you don't like this PR?
